Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.h

```diff
@@ -91,7 +97,8 @@ class SkylanderPortalWindow : public QWidget
   QPushButton* m_path_select;
 
   std::array<QCheckBox*, 5> m_game_filters;
-  std::array<QRadioButton*, 10> m_element_filter;
+  std::array<QRadioButton*, 12> m_element_filter;
```

Comment:
Here you can use the constexpr for each of the different lists, mentioned previously